### PR TITLE
Adds/changes some tips.

### PR DESCRIPTION
**Pull request by RandomMarine**
No further comment or changelog really necessary.

### CODE REVIEW

File: strings/tips.txt [hunk 1]
```diff
@@ -19,14 +19,15 @@ Maintenance is full of equipment that is randomized every round. Look around and
...
 As the Captain, you can purchase a new emergency shuttle using a communications console. Some require credits, while others give you credits in exchange. Keep in mind that purchasing dangerous shuttles will incur the ire of your crew.
 As the Chief Medical Officer, your hypospray is like a refillable instant injection syringe that can hold 30 units as opposed to the standard 15.
 As the Chief Medical Officer, coordinate and communicate with your doctors, chemists, and geneticists during a nuclear emergency, blob infestation, or some other crisis to keep people alive and fighting.
 As a Medical Doctor, you can attempt to drain blood from a husk with a syringe to determine the cause. If you can extract blood, it was caused by extreme temperatures or lasers, if there is no blood to extract, you have confirmed the presence of changelings.
-As a Medical Doctor, charcoal will not only heal toxin damage dealt by poisons, but will actively remove them.
+As a Medical Doctor, while both heal toxin damage, the difference between charcoal and antitoxin is that charcoal will actively remove all other reagents from one's body, while antitoxin only removes various toxins - but can overdose.
```
Comment (Anonmare) [inline strings/tips.txt]: This isn't true, anti-toxin is a purgative, just like Charcoal

Comment (RandomMarine) [reply strings/tips.txt]: https://github.com/tgstation/tgstation/blob/master/code/modules/reagents/chemistry/reagents/medicine_reagents.dm#L994
But it is

File: strings/tips.txt [hunk 2]
```diff
@@ -19,14 +19,15 @@ Maintenance is full of equipment that is randomized every round. Look around and
 Some roles cannot be antagonists by default, but antag selection is decided first. For instance, you can set Security Officer to High without affecting your chances of becoming an antag -- the game will just select a different role.
 There are many places around the station to hide contraband. A few for starters: linen boxes, toilet cisterns, body bags. Experiment to find more!
 On some maps, you can use a machine in the vault to deposit space cash for cargo points or rob cargo blind.
+If you suspect a container to be booby-trapped with a mousetrap-rigged grenade, you can safely dump its contents by clickdragging it onto an open tile or table.
```
Comment (Iamgoofball) [inline strings/tips.txt]: this is an exploit tbh

**Comment by 81Denton** [comment]
Could you add this one as well?

`As the Chief Engineer, the power flow control console in your office lists details about all APCs and lets you control them remotely.`

File: strings/tips.txt (hunk 1 above)
Comment (optimumtact) [reply strings/tips.txt]: blame goof

File: strings/tips.txt (hunk 2 above)
Comment (optimumtact) [reply strings/tips.txt]: good luck fixing it

Comment (ExcessiveUseOfCobblestone) [reply strings/tips.txt]: If we acknowledge it's an exploit and not a exploit-elevated-into-feature we probably shouldn't put it as a tip regardless of how much of a pain it'll be to fix it.

Comment (ShizCalev) [reply strings/tips.txt]: Yeah this is a pretty obvious exploit. Remove this please.

Comment (ShizCalev) [inline strings/tips.txt]: Yeah this is a pretty obvious exploit. Remove this please.

**Review by ShizCalev — CHANGES_REQUESTED** [review]

**Pull request merged**